config/app.default.php

```diff
@@ -238,7 +242,7 @@
         ],
```

Comment:
Should Email.default.charset and Email.default.headerCharset not be set to utf-8 by default?

Reply:
Same.

Reply:
ok